test/unit/utScaleProcess.cpp

```diff
@@ -69,18 +69,5 @@ TEST_F( utScaleProcess, accessScaleTest ) {
     EXPECT_FLOAT_EQ( 2.0f, process.getScale() );
 }
 
```

Comment:
Scale tests no longer can detect scale value because the scale is never modified the original transforms are overwritten with the scale instead of modifying the scale, because the scale tended to not be ideal as models with 25,25,25 scale would be translated to 0.025, 0.025, 0.025 which caused more problems.

The best method I found through multiple experiments was to modify the 'translation' rather than the scale it also produced the best results.

Reply:
Good point, didn't noticed that during my review.